udp/src/main/scala/akka/stream/alpakka/udp/scaladsl/Udp.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.udp.scaladsl
+
+import java.net.InetSocketAddress
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.stream.alpakka.udp.UdpMessage
+import akka.stream.alpakka.udp.impl.{UdpBindFlow, UdpFireAndForgetFlow}
+import akka.stream.scaladsl.Sink
+import akka.stream.scaladsl.Flow
+
+import scala.concurrent.Future
+
+object Udp {
+
+  /**
+   * Creates a flow that will send all incoming [UdpMessage] messages to the remote address
+   * contained in the message. All incoming messages are also emitted from the flow for
+   * subsequent processing.
+   */
+  def fireAndForgetFlow()(implicit sys: ActorSystem): Flow[UdpMessage, UdpMessage, NotUsed] =
+    Flow.fromGraph(new UdpFireAndForgetFlow())
+
+  /**
+   * Creates a sink that will send all incoming [UdpMessage] messages to the remote address
+   * contained in the message.
+   */
+  def fireAndForgetSink()(implicit sys: ActorSystem): Sink[UdpMessage, NotUsed] = fireAndForgetFlow().to(Sink.ignore)
+
+  /**
+   * Creates a flow that upon materialization binds to the given `localAddress`. All incoming
+   * messages to the `localAddress` are emitted from the flow. All incoming messages to the flow
+   * are sent to the remote address contained in the message.
+   */
+  def bindFlow(
+      localAddress: InetSocketAddress
+  )(implicit sys: ActorSystem): Flow[UdpMessage, UdpMessage, Future[InetSocketAddress]] =
+    Flow.fromGraph(new UdpBindFlow(localAddress))
+}
```

Comment:
Why not just send/bind as names?

Reply:
@huntc this story hangs on github for so long that I've completely forgot why I used such names. It probably is a good idea but I'm wondering what's the point in investing more time into this while there is zero pick up from the maintainers to get this in.

That's not to say there's no will to improve, the reason why I've abandoned work on this was because there was no indication from the maintainers on the desired direction. If there is the will to get this in, one will find the time to resolve review requests.

Reply:
Hi @radekg. This PR is an indication of wanting to get this in. :)

@huntc having "flow"/"sink" in the factory methods is a pattern that has been emerging in various Alpakka connectors. It is especially useful when there is a need to differentiate a factory based on the return type but which takes exactly the same arguments.

However I aggree that "fireAndForget" might be a bit too verbose. Having "sendFlow", "sendSink" and "bindFlow" sounds good to me.

Reply:
I have no problems with such modifications. Apologies if my communication from yesterday sounded harsh.